README.md

```diff
@@ -117,29 +118,26 @@ List of content
 * [Blockchain2Graph](https://github.com/straumat/blockchain2graph) Graph Database Builder
 
 # Full nodes
-* [Bitcoinjs-server](https://github.com/bitcoinjs/bitcoinjs-server) Bitcoin network node in JavaScript using Node.js / LevelDB
 * [Bitcoin-ruby-node](https://github.com/mhanne/bitcoin-ruby-node) bitcoin node based on bitcoin-ruby-blockchain
-* [Toshi by Coinbase](https://toshi.io/) An open source Bitcoin node built to power large scale web applications
-* [Fullnode](https://github.com/ryanxcharles/fullnode) Javascript implementation of bitcoin
-* [Insight API](https://github.com/bitpay/insight-api) A bitcoin blockchain API for web wallets
-* [Bitcoind.JS](https://github.com/bitpay/bitcoind.js) bitcoind linked to node.js by BitPay
+* [Fullnode](https://github.com/ryanxcharles/yours-bitcoin) Javascript implementation of bitcoin
```

Comment:
isnt that just a fork of https://github.com/moneybutton/yours-bitcoin  ?

Reply:
Maybe. I just changed old link on new.